src/wp-includes/blocks.php

```diff
@@ -283,6 +283,17 @@ function get_block_metadata_i18n_schema() {
  * @return WP_Block_Type|false The registered block type on success, or false on failure.
  */
 function register_block_type_from_metadata( $file_or_folder, $args = array() ) {
+	/**
+	 * Get an array of metadata from a PHP file.
+	 * This improves performance for core blocks as it's only necessary to read a single PHP file
+	 * instead of reading a JSON file per-block, and then decoding from JSON to PHP.
+	 * Using a static var ensures that the metadata is read only once per request.
+	 */
+	static $core_blocks_meta;
+	if ( ! $core_blocks_meta ) {
+		$core_blocks_meta = include( ABSPATH . WPINC . '/blocks/blocks-json.php' );
```

Comment:
```suggestion
		$core_blocks_meta = file_get_contents( ABSPATH . WPINC . '/blocks/blocks-json.php' );
```
Also, tests seem to pass when this change is made.

Reply:
What about changing this to `include_once`? `file_get_contents()` should be avoided as much as possible.

Reply:
Ah, woops. `file_get_contents` is incorrect here. I missed that the file in question immediately returns a value.

Unfortunately, tests still fail with `include_once`.

Also, whether `include` or `include_once` is used here, note that [parentheses should not be used](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#writing-include-require-statements).

Reply:
Thanks! Removed the parentheses and changed `include` to `include_once` 👍